Azure/sonic-buildimage
# [device][as7816-64x]Update fan speed reading.

Signed-off-by: roy_lee <roy_lee@accton.com>

- Why I did it
Original converting from register is wrong, it makes the fan speed much higher than it is.

- How I did it
Change the way fan speed is calculated from CPLD.

- How to verify it
sensors
or check /sys/bus/i2c/devices/17-0068/fan*_input

- Which release branch to backport (provide reason below if seleted)
It's a driver updating and only changes on few numbers. The change should conform to versions of kernel .

[o] 201811
[o] 201911
[o] 202006

- Description for the changelog
As the code diff.
